src/renderer.cpp

```diff
@@ -1366,3 +2001,151 @@ uint16_t Renderer::getInternalCoordY(uint16_t inY)
 {
     return (inY * framebufferHeight) / game_height;
 }
+
+void lighting_debug(bool* isOpen)

```

Comment:
This code shhould not be part of this class, as you notice for DevTools we have separate files containing this code. Cosinder moving this to a dedicated `.cpp` file and separate class named as you prefer ( `light.cpp` maybe? )

Reply:
I added `lighting_debug.h` and  `lighting_debug.cpp` and moved this code there.
I also modified the code so that it is only called for FF7.